clients/drcachesim/tests/irregular-windows-simple.templatex

```diff
@@ -0,0 +1,17 @@
+Hit delay threshold: enabling tracing.
+Hit tracing window #0 limit: disabling tracing.
+Hit retrace threshold: enabling tracing for window #1.
+Hit tracing window #1 limit: disabling tracing.
+Hit retrace threshold: enabling tracing for window #2.
+Hit tracing window #2 limit: disabling tracing.
+.*
+---- <application exited with code 0> ----
+Basic counts tool results:
+.*
+Total windows: [0-9]*
+Window #0:
+.*
```

Comment:
Should also verify that the count of instrs in each window matches the specified values.

Reply:
I took that from windows-simple.templatex, which does not specify instruction counts for the windows. I assumed because the counts might vary slightly from run to run, but I haven't checked. Will check and put the expected instruction count if it doesn't change.

Reply:
It may be so. We could try to add a regexp check that verifies the higher order bits atleast.

Reply:
Added higher order bits to the expected #instr per window. Done.

Reply:
Actually, I started seeing quite a bit of variation depending on the platform (especially x86 vs aarch64), so I had to remove these checks.

Reply:
Note: this variation is still within range (around +5K instructions, since we don't support exact count). We could try to use bigger windows, but simple-app is relatively short.

Reply:
Done in `clients/drcachesim/tests/irregular-windows-simple.templatex`.